### ISSUE TITLE
new field type or extended url field type

### ISSUE THREAD

**Issue by rallek** [Guite/MostGenerator#514]
we have the flield `url`. This demands a full url. But if I want to have a relative path like `/myFolder/index.html` this field is not working. Now I will use a string field without any validation (in the current usecase it is ok). Please create a new field type for `path` or make it an option for `url`

This new field should also look if the path is available (as an option).

To make it perfect: If the path is a folder and the folder do not exist a dialog should ask if a new folder should be created (if allowed). Again, this should be optional.

**Comment by Guite** [comment]
Depends on #97

**Comment by Guite** [comment]
@rallek can you please explain the use case you'd like to have this for?

**Comment by Guite** [comment]
What you are looking for are not pathes but relative links. These can not be validated.

An option for relative pathes would be possible though, but this would be only interesting for very rare use cases (e.g. a web hosting management panel).

Closing as "won't fix".
